client/src/components/Rooms/Rooms.scss

```diff
@@ -1,9 +1,10 @@
 .rooms-page {
-    background-color: $black;
+    background-color: $white;
+    text-align: center;
 
     &-cards {
-        width: 90%;
-        margin: 0 auto;
+        width: 90% !important;
+        margin: 0 auto !important;
```

Comment:
Why is important used here? #root ...selector helps set desired width?

Reply:
fixed it...